user_guide/business_intelligence.rst

```diff
@@ -0,0 +1,23 @@
+.. _business-intelligence:
+
+Business Intelligence Tools
+===========================
+
+OroCRM provides a set of functionalities for data analysis and decision-making. With numerous dedicated widgets the users can 
+easily access, analyse all the necessary information and keep the track of the important changes and updates, reports 
```

Comment:
easily access and analyze all the necessary information in order to keep track of any important changes and updates. For instance, reports